pipelines/rj_segur__forca_municipal/client.py

```
@@ -0,0 +1,205 @@
+        self._username: Optional[str] = env.API_FORCA_MUNICIPAL__API_LOGIN
+        self._password: Optional[str] = env.API_FORCA_MUNICIPAL__API_PASSWORD
+        self._token: Optional[str] = None
+        self._lock: asyncio.Lock = asyncio.Lock()
+        self._client: Optional[httpx.AsyncClient] = None
+
+    async def __aenter__(self) -> "FMApi":
+        self._client = httpx.AsyncClient(
+            limits=httpx.Limits(max_connections=20, max_keepalive_connections=10),
+            proxy=self._proxy_url,
+            timeout=httpx.Timeout(API_TIMEOUT, connect=API_CONNECT_TIMEOUT),
+            verify=False,
```

Ruff (0.15.12)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)